cscs-checks/microbenchmarks/gpu/gpu_burn/gpu_burn_test.py

```diff
@@ -8,123 +8,49 @@
 import reframe as rfm
 import reframe.utility.sanity as sn
 
+from library.microbenchmarks.gpu.gpu_burn import GpuBurnBase
+import cscslib.microbenchmarks.gpu.hooks as hooks
```

Comment:
One question that the hook name rises is whether how one classify hooks. I am referring to classical distinction of  `hooks.gpu` vs `gpu.hooks`... What do you think?